<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerAccount</class>
 <widget class="QWidget" name="registerAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>491</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="passwordAgainLabel">
      <property name="font">
       <font>
        <family>AaJHNDMCC (Non-Commercial Use)</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>再次输入密码</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="font">
       <font>
        <family>AaJHNDMCC (Non-Commercial Use)</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="usernameLabel">
      <property name="font">
       <font>
        <family>AaJHNDMCC (Non-Commercial Use)</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="usernameInput">
      <property name="maxLength">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="passwordInput">
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="passwordAgainInput">
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>360</y>
     <width>112</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AaJHNDMCC (Non-Commercial Use)</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>540</y>
     <width>112</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AaJHNDMCC (Non-Commercial Use)</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="interfaceNameLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AaJHNDMCC (Non-Commercial Use)</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册新账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Kaiti SC</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
